addon/components/polaris-button/link.js

```diff
@@ -4,26 +4,18 @@ import mapEventToAction from '../../utils/map-event-to-action';
 
 export default BaseComponent.extend({
   tagName: 'a',
-  attributeBindings: [
-    'url:href',
-    'dataPolarisUnstyled:data-polaris-unstyled',
-    'target',
-    'rel',
-  ],
+  attributeBindings: ['url:href', 'dataPolarisUnstyled:data-polaris-unstyled', 'target', 'rel'],
 
   dataPolarisUnstyled: 'true',
 
+  //  Allow click to perform its default action.
+  click: mapEventToAction('onClick', { preventDefault: false }),
```

Comment:
This is an action handler so should really come after computed properties...

Reply:
ESLint isn't quite smart to handle this. But I could add it to our plugin if we feel strongly about this. Also how about we extract the util to its own own addon?

Reply:
No real need to move it to its own addon IMO - certainly not at this point anyway. It's a small thing and doesn't see much usage even in `ember-polaris` 🤷‍♂️